Added more Georgian keyboards

dracid opened the pull request:
1) Qwerty-based no-shift layout
2) Ergonomic
3) Typewriter (classic)

tenextractor commented:
I would not recommend adding 12 keys to a single keyboard row, this makes it overcrowded, see https://github.com/florisboard/florisboard/issues/2092 . Instead, we can shift one of the keys in that row to the bottom row.

Example for ergonomic layout:
![image](https://github.com/user-attachments/assets/5831105f-57fc-45a0-915b-77e70d319c4a)
YAML:
```
name: Georgian Ergonomic
languages: ka_GE
minimumFunctionalKeyWidth: 0.0
rows:
  - letters:
    - 'ჩ'
    - 'პ'
    - 'უ'
    - 'ძ'
    - 'ტ'
    - 'თ'
    - 'ნ'
    - 'ვ'
    - 'შ'
    - 'კ'
    - 'ქ'
  - letters:
    - 'ხ'
    - 'ი'
    - 'ა'
    - 'ე'
    - 'ო'
    - 'დ'
    - 'მ'
    - 'ს'
    - 'რ'
    - 'ბ'
    - 'გ'
  - letters:
    - 'ჯ'
    - 'ჰ'
    - 'ყ'
    - 'ღ'
    - 'ჟ'
    - 'ზ'
    - 'ც'
    - 'ლ'
    - 'ფ'
    - 'წ'
    - $delete
  - bottom:
    - $symbols
    - 'ჭ'
    - $action
    - $space
    - $contextual
    - '.'
    - $enter

```

Example for NoShift layout:
![image](https://github.com/user-attachments/assets/76cac51c-d52c-404f-afb1-81abe61fcdd9)
YAML:
```
name: Georgian (MES)
languages: ka_GE
minimumFunctionalKeyWidth: 0.0
rows:
  - letters:
    - 'ქ'
    - 'წ'
    - ['ე', '%']
    - 'რ'
    - 'ტ'
    - ['ყ', '%', 'ჸ']
    - 'უ'
    - ['ი', '%', 'ჲ']
    - 'ო'
    - 'პ'
    - 'შ'
  - letters:
    - ['ა', 'ჺ']
    - 'ს'
    - 'დ'
    - ['ფ', 'ჶ']
    - ['გ', 'ჹ']
    - ['ჰ', 'ჱ']
    - ['ჯ', 'ჷ']
    - 'კ'
    - 'ლ'
    - 'თ'
    - 'ჩ'
  - letters:
    - 'ჟ'
    - ['ზ', 'ჵ']
    - ['ხ', 'ჴ']
    - 'ც'
    - ['ვ', 'ჳ']
    - 'ბ'
    - ['ნ', 'ჼ']
    - 'მ'
    - 'ძ'
    - 'ღ'
    - $delete
  - bottom:
    - $symbols
    - 'ჭ'
    - $action
    - $space
    - $contextual
    - '.'
    - $enter
```

Another NoShift layout (cross compatible with default shift layout, so can be used to replace it)
![image](https://github.com/user-attachments/assets/d108d5ca-209e-4893-94e2-385c459b2d00)
YAML:
```
name: Georgian QWERTY
languages: ka_GE
attributes: {shiftable: false} # Automatic shifting breaks many of the characters
rows:
  - letters:
    - 'ჭ'
    - 'ღ'
    - 'თ'
    - 'შ'
    - 'ჟ'
    - 'ძ'
    - 'ჩ'
  - letters:
    - 'ქ'
    - {type: case, normal: ['წ', 'ჭ'], shiftedManually: ['ჭ']}
    - ['ე', '%', 'ჱ']
    - {type: case, normal: ['რ', 'ღ'], shiftedManually: ['ღ']}
    - {type: case, normal: ['ტ', 'თ'], shiftedManually: ['თ']}
    - ['ყ', '%', 'ჸ']
    - 'უ'
    - ['ი', '%', 'ჲ']
    - 'ო'
    - 'პ'
  - letters:
    - ['ა', 'ჺ']
    - {type: case, normal: ['ს', 'შ'], shiftedManually: ['შ']}
    - 'დ'
    - ['ფ', 'ჶ']
    - ['გ', 'ჹ']
    - ['ჰ', 'ჵ']
    - {type: case, normal: ['ჯ', 'ჟ', 'ჷ'], shiftedManually: ['ჟ']}
    - 'კ'
    - 'ლ'
  - letters:
    - {type: case, normal: ['ზ', 'ძ'], shiftedManually: ['ძ']}
    - ['ხ', 'ჴ']
    - {type: case, normal: ['ც', 'ჩ'], shiftedManually: ['ჩ']}
    - ['ვ', 'ჳ']
    - 'ბ'
    - ['ნ', 'ჼ']
    - 'მ'
```

Example for typewriter layout:
![image](https://github.com/user-attachments/assets/e9fe7a2b-f5a6-428e-9e74-37633b0757b4)
YAML:
```
name: Georgian Typewriter
languages: ka_GE
minimumFunctionalKeyWidth: 0.0
rows:
  - letters:
    - 'ღ'
    - 'ჯ'
    - 'უ'
    - 'კ'
    - ['ე', 'ჱ']
    - ['ნ', 'ჼ']
    - ['გ', 'ჹ']
    - 'შ'
    - 'წ'
    - 'ზ'
    - ['ხ', 'ჴ']
  - letters:
    - ['ფ', 'ჶ']
    - 'ძ'
    - ['ვ', 'ჳ']
    - 'თ'
    - ['ა', 'ჺ']
    - 'პ'
    - 'რ'
    - 'ო'
    - 'ლ'
    - 'დ'
    - 'ჟ'
  - letters:
    - 'ჭ'
    - 'ჩ'
    - ['ყ', 'ჸ']
    - 'ს'
    - 'მ'
    - ['ი', 'ჲ']
    - 'ტ'
    - 'ქ'
    - 'ბ'
    - ['ჰ', 'ჵ']
    - $delete
  - bottom:
    - $symbols
    - 'ც'
    - $action
    - $space
    - $contextual
    - '.'
    - $enter
```

dracid commented:
Hello group,

I have now updated the layouts to include a 3-row and a 4-row shiftless layout.
3-row is somewhat standard (been used for years in Anysoft Keyboard, but now abandoned)
4-row is new as per suggestions in this thread. I am actually quite liking these!!

Also, I have added a MTAVRULI style of Georgian letters (as per [here](https://en.wikipedia.org/wiki/Georgian_Extended)).
So, in total we have the following keyboard layouts:
1) QWERTY classic (modern layout everyone uses, already in FUTO)
2) Ergonomic (standard ka_GE, but rarely used)
3) Typewriter (also standard but from legacy typewriters)
4) Shiftless QWERTY a) 3-row and b) 4-row
5) Shiftless QWERTY MTAVRULI Style a) 3-row and b) 4-row

The shiftless QWERTY layouts will be the basis of future development of the Swipe-style typing in FUTO.

Samsung Android Keyboard has swipe, but it does NOT work properly. This is due to 6 (ჭ, ღ, თ, შ, ჟ, ძ, ჩ)  common symbols requiring SHIFT, so the swipe algorithm (which is also closed source Samsung code) has no idea how to type many words.
